diff --git a/core/jni/fd_utils-inl.h b/core/jni/fd_utils-inl.h
index a68a60274ac..c6fc5088577 100644
--- a/core/jni/fd_utils-inl.h
+++ b/core/jni/fd_utils-inl.h
@@ -58,6 +58,7 @@ static const char* kPathWhitelist[] = {
   "/dev/urandom",
   "/dev/ion",
   "/dev/dri/renderD129", // Fixes b/31172436
+  "/proc/ged", // MTK add
   "/system/framework/org.cyanogenmod.platform-res.apk",
 #ifdef PATH_WHITELIST_EXTRA_H
 PATH_WHITELIST_EXTRA_H
